Makefile

```diff
@@ -76,7 +76,7 @@ deploy: manifests ## Deploy controller in the configured Kubernetes cluster in ~
 
 .PHONY: docker-build
 docker-build: ## Build the docker image
-	docker image inspect ${IMG} && echo "docker image ${IMG} already built" || ${DOCKER} build . -t ${IMG}
```

Comment:
This was skipping the build on code change. Docker already caches the layers. Ok for you to remove this check?

Reply:
Reverted